fetch_gazebo_demo/scripts/demo.py

```diff
@@ -149,7 +149,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
+                                         use_service = False)
```

Comment:
I think you actually want to just remove the "wait = False" -- the new service interface is quite speedy and very reliable (unlike the old "wait for sync").